src/services/user.service.js

```diff
@@ -66,6 +66,19 @@ const updateUserById = async (userId, updateBody) => {
   return user;
 };
 
+const addStudentInformation = async (userId, updateBody) => {
+  const user = await getUserById(userId);
+  if (!user) {
+    throw new ApiError(httpStatus.NOT_FOUND, 'User not found');
+  }
+  if (!updateBody.age && !updateBody.hobby) {
+    throw new ApiError(httpStatus.BAD_REQUEST, 'Email already taken');
+  }
+  Object.assign(user, updateBody);
```

Comment:
```suggestion
  Object.assign(user, { age: updateBody.age, hobby: updateBody.hobby });
```
to make sure the student updates only this information you should just update age and hobby